### PR TITLE
GR CEM 2581 - Profile Modal

### PR DESCRIPTION
**Pull request by mmendoza1964**
Created a Profile Modal component that displays a user's profile picture, name, tags, and the date the account was created.
The tags are contained in a tag group container and displayed in their own row.
Each of the modal elements are displayed in their own row, so that it is easily viewable on mobile platforms.
There is some margin and padding styling around the elements. The section labels for the tags and date are bold to make the section labels and the user info information more distinct.

profileModal-ex1

### CODE REVIEW

File: src/Containers/ProfileModal/ProfileModal.tsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    return (
+        <MyModal state={[true, () => true]} {...props}>
+            <ProfileImage 
+                src={image}/>
+            <Text>{name}</Text>
+            <TextBold>Tags</TextBold>
+            {displayTags(tags)}
+            <TextBold>Date Created</TextBold>
+            <TextSmall>{date}</TextSmall>
+
```
Comment (ralph-dev) [inline src/Containers/ProfileModal/ProfileModal.tsx:45]: Remove line

File: src/Containers/ProfileModal/ProfileModal.tsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+     */
+    const displayTags = (tagComponents: Array<TagProps>) => 
+        tagComponents.map((tag) => (
+            <div>
+                <MyTag {...tag} />
+            </div>
+        ));
+
+    return (
+        <MyModal state={[true, () => true]} {...props}>
```
Comment (ralph-dev) [inline src/Containers/ProfileModal/ProfileModal.tsx:37]: With state like this the model will always show in the application

File: src/Containers/ProfileModal/ProfileModal.tsx [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+        font-size: ${theme.font.size.default};
+    `}
+    margin: 2px;
+`;
+
+const ProfileImage = styled.img`
+    width: 80px;
+    height: 80px;
+    border-radius: 999px;
+    background-color: grey;
```
Comment (ralph-dev) [inline src/Containers/ProfileModal/ProfileModal.tsx:86]: Theme

**Review by ralph-dev — CHANGES_REQUESTED** [review]

**Review by ralph-dev — CHANGES_REQUESTED** [review]
Also include a picture of a story where it's compressed, mobile

**Comment by mmendoza1964** [comment]
Modal resizes components to fit view size
mobile view example (small mobile
profileModal-mobileSM
)

**Review by ralph-dev — APPROVED** [review]
Lgtm

**Pull request merged**